### DISCUSSION TITLE
Popup Close Button

### DISCUSSION THREAD

**Discussion by t0ben** [elementor/elementor#21773] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

The Popup close Button is still not changeable.

### Describe the solution you'd like

Size/Color and optionally SVG

### Additional context

Already mentioned in closed "Allow Custom Icon for Popup Close Button #9177" and merged with https://github.com/elementor/elementor/issues/11361, which is also closed - the issue is still not resolved though.

**Comment by hitsgee** [comment]
Would be also amazing to have a background feature that works like the icon widget. Stacked for example.
